src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._disabled;
   }
 
+  /**
+   * Indicated whether to hide the flag in the input element.
+   * @default "false"
```

Comment:
Is the default value supposed to be in quote marks? I might have misunderstood @Blackbaud-SteveBrush's instructions on this, but I thought `string` properties needed single quote marks and that other types didn't need quote marks.

Reply:
I had just followed one of the other components in the library's example as I haven't done a lot of these. I removed the quotes and things seem to work; so if you think they don't need to be there then I'll leave them out :)